programs/grabberControls2Gui/run.py

```diff
@@ -23,13 +25,33 @@
 @begin.start(auto_convert=True)
 @begin.logging
 def main(remote_port: 'Remote port running the AravisGigE grabber'='/grabber'):
+    # Check for YARP network
+    yarp.Network.init()
+    if not yarp.Network.checkNetwork():
+        logging.error('Could not connect to YARP network. Please try running YARP server.')
+        quit()
```

Comment:
Further down you use `sys.exit()`. Can we stick to only one of them?

Reply:
Done at https://github.com/roboticslab-uc3m/yarp-devices/commit/3c1f7f367dd637523dfe65b3335a516d5ca9d4ef !

Reply:
Thanks!